doc/tutorials/basic/pn_panel.md

```diff
@@ -0,0 +1,440 @@
+)
+component = pn.panel(data)
+print(component)
+component.servable()
+```
+
+```{note}
+If we want to display larger dataframes, customize the way the dataframes are displayed, or make them more interactive, we can find specialized components in the [Component Gallery](../../reference/index.md) supporting these use cases. For example, the [Tabulator](../../reference/widgets/Tabulator.ipynb) widget and [Perspective](../../reference/panes/Perspective.ipynb) pane.
+```
+
+## Display any Python object
+
+`pn.panel` can display (almost) any Python object.
+
+Run the code below
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+component = pn.Column(
+    pn.panel({"Wind Speeds": [0, 3, 6, 9, 12, 15, 18, 21], "Power Output": [0,39,260,780, 1300, 1300, 0, 0]}),
+    pn.panel("https://assets.holoviz.org/panel/tutorials/wind_turbine.png", height=100),
+    pn.panel("https://assets.holoviz.org/panel/tutorials/wind_turbine.mp3"),
+)
+print(component)
+
+component.servable()
+```
+
+## Display any Python object in a layout
+
+Run the code below
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+component = pn.Column(
+    pn.panel({"Wind Speeds": [0, 3, 6, 9, 12, 15, 18, 21], "Power Output": [0,39,260,780, 1300, 1300, 0, 0]}),
+    "https://assets.holoviz.org/panel/tutorials/wind_turbine.png",
+    "https://assets.holoviz.org/panel/tutorials/wind_turbine.mp3",
+)
+print(component)
+
+component.servable()
+```
+
+:::{note}
+When Python objects are given as an argument to a Panel [Layout](https://panel.holoviz.org/reference/index.html#layouts) like [`pn.Column`](../../reference/layouts/Column.ipynb), then `pn.Column` will automatically apply `pn.panel` to the objects for you.
+:::
+
+Please notice that the image of the dice is very tall. To fine-tune the way it is displayed, we can use `pn.panel`.
+
+Run the code below
+
+```{pyodide}
+import panel as pn
+
+pn.extension()
+
+component = pn.Column(
+    pn.panel({"Wind Speeds": [0, 3, 6, 9, 12, 15, 18, 21], "Power Output": [0,39,260,780, 1300, 1300, 0, 0]}),
+    pn.panel("https://assets.holoviz.org/panel/tutorials/wind_turbine.png", height=100),
+    pn.panel("https://assets.holoviz.org/panel/tutorials/wind_turbine.mp3", styles={"background": "orange", "padding": "10px"}),
+)
+print(component)
+
+component.servable()
+```
+
+:::{note}
+The example above sets the *css* `styles` of the `Audio` player. The `styles` parameter is introduced in the [Styles](style.md) tutorial.
+:::
+
+## Consider Performance
+
+:::{note}
+`pn.panel` is a versatile helper function that converts objects into a [*Pane*](https://panel.holoviz.org/reference/index.html#panes). It automatically selects the best *representation* for an object based on available [*Pane*](https://panel.holoviz.org/reference/index.html#panes) types, ranking them by priority.
+
+For optimal performance, specify the desired *Pane* type directly, like `pn.pane.Matplotlib(fig)` instead of using `pn.panel(fig)`.
```

Comment:
I sort of agree. I've been back and forth on this because according to Diataxis tutorials are *learning by doing* without explanations. So I believe in its world its ok to just use concepts without explanations.

I've added the text "You will learn about *Panes*  in the [Display Content with Panes](panes.md) section.". Let me know if I can do something even better.

Reply:
> I've been back and forth on this because according to Diataxis tutorials are learning by doing without explanations. So I believe in its world its ok to just use concepts without explanations.

Yeah Diataxis is a bit confusing on this front. But my feedback, after doing the Django Tutorial and reviewing Panel one, is that some explanation is *required* in a tutorial:

- you can't just let people figure out everything on their own, some things are obvious but some are not at all
- explanations help build a good flow, you do something and then you reflect on it. Without explanations, it's just a series of actions to do, and then you end up mechanically running code without really getting what's going on
- you don't need to explain everything in depth (I believe that's what Diataxis tries to steer you away from), light explanations are fine though in my opinion

Reply:
I've removed the code example.